open_samus_returns_rando/__init__.py

```diff
@@ -1 +1 @@
-from .samus_returns_patcher import patch
```

Comment:
I don't think we need this import at all.

Reply:
I just copied from Dread lol